### PR TITLE
Add Brookings File

### PR DESCRIPTION
**Pull request by ling0015**
This is the beginning of the brookings file.

Check it out.

### CODE REVIEW

File: ling0015-brookings.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Brookings SD
+
+### Restaurants
+
+### Local Attractions
+- Children's Museum
```
Comment (NGenetzky) [inline ling0015-brookings.md]: Wow, your actually typing everything he puts?

File: ling0015-brookings.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Brookings SD
```
Comment (KimThomp) [inline ling0015-brookings.md]: Hey @ling0015 I have not been to Brookings, SD is it close to a beach?

**Comment by ling0015** [comment]
@KimThomp Nah it is in the middle of the plains, no beaches, no trees just a lot of grass.

**Comment by ling0015** [comment]
@bodenhamerj Hey like the remote attractions I added? a93edee

**Comment by NGenetzky** [comment]
Muhaha I steal [stuff ](https://github.com/githubteacher/developers-july-PM/tree/Brookings%2CSD)

**Comment by ling0015** [comment]
is that how these emojis work?

**Pull request merged**